### DISCUSSION TITLE
Pro Plugin

### DISCUSSION THREAD

**Discussion by celltower** [gdarko/digital-license-manager#43] [General]
Hey do I have to still keep installed and have the Free Version of the Digital License Plugin activated to use all functions?

**Comment by celltower** [comment]
I have deleted the Free Version of the Plugin and any thing works but I can not find the Licenses add and edit Templates but the add licenses and edit functionality still work

**Comment by gdarko** [comment]
The FREE version is optional if you are running the PRO version.

The FREE version does not fully load when using the PRO version. 

The FREE version is bundled into the PRO version, you can see the vendor folder.

The only use case that we recommend to run both versions is if you want to translate digtial-lciense-manager free version while also running the PRO version. If you want to translate the free version, WordPress requires the digital-license-manager plugin to be active, this is limitation.

**Reply by celltower** [reply]
mhmm Okay do you have any Idee why I can see the files in my Hosting File Manger but in the Plugin files Editor not? The Permissions  are set normal
